pkg/dao/repositories.go

```diff
@@ -5,52 +5,54 @@ import (
 	"gorm.io/gorm"
 )
 
-type PublicRepository struct {
+// Repository internal (non-user facing) representation of a repository
+type Repository struct {
 	UUID     string
 	URL      string
 	Revision string
 }
 
-func GetPublicRepositoryDao(db *gorm.DB) PublicRepositoryDao {
-	return publicRepositoryDaoImpl{
+func GetRepositoryDao(db *gorm.DB) RepositoryDao {
+	return repositoryDaoImpl{
 		db: db,
 	}
 }
 
-type publicRepositoryDaoImpl struct {
+type repositoryDaoImpl struct {
 	db *gorm.DB
 }
 
-func (p publicRepositoryDaoImpl) FetchForUrl(url string) (error, PublicRepository) {
+func (p repositoryDaoImpl) FetchForUrl(url string) (error, Repository) {
 	repo := models.Repository{}
 	result := p.db.Where("URL = ?", url).First(&repo)
 	if result.Error != nil {
-		return result.Error, PublicRepository{}
+		return result.Error, Repository{}
 	}
-	return nil, PublicRepository{
+	return nil, Repository{
 		UUID:     repo.UUID,
 		URL:      repo.URL,
 		Revision: repo.Revision,
 	}
 }
 
-func (p publicRepositoryDaoImpl) List() (error, []PublicRepository) {
+func (p repositoryDaoImpl) List() (error, []Repository) {
 	var repos []models.Repository
-	var publicRepos []PublicRepository
-	result := p.db.Where("public = true").Find(&repos)
+	var publicRepos []Repository
```

Comment:
I think we should try to remove the name `publicRepos` since it's not dealing with public repos anymore. Maybe `internalRepos`? Or rename `repos` to `modelRepos` and `publicRepos` to `repos`? Not sure.

Reply:
went with dbRepos and repos